posthog/tasks/status_report.py

```diff
@@ -0,0 +1,19 @@
+import logging
+
+import posthoganalytics
+from django.utils import timezone
+
+from posthog.models import Event
+from posthog.utils import get_machine_id
+
+logger = logging.getLogger(__name__)
+
+
+def status_report() -> None:
+    period_end = (timezone.now() - timezone.timedelta(timezone.now().weekday())).replace(
+        hour=0, minute=0, second=0, microsecond=0
+    )  # very start of the current Monday
+    period_start = period_end - timezone.timedelta(7)  # very start of the Monday preceding the current one
+    events_considered = Event.objects.filter(created_at__gte=period_start, created_at_lt=period_end)
+    report = {"period": [period_start.isoformat(), period_end.isoformat()], "event_count": events_considered.count()}
+    posthoganalytics.capture(get_machine_id(), "instance status report", report)
```

Comment:
A few points here,
- I think it might be a bit better to do a team-based report in case the instance has multiple teams and just use the machine ID as a property to relate them to a single instance, that way we can do analyses on a team-scope or instance-scope.
- We need a way to relate these teams/machines to specific users, as we do analyses on a user-basis.
- I believe it'd be useful to add some additional metadata information on the events captured like,
    - The number of events coming from each library.
    - The number of events using autocapture.
    - The number of active persons / users.
    - The version of PostHog being used.

Reply:
Yep, absolutely, this was just an initial version. I now added PostHog version, a breakdown of events by event name (which is useful for analyzing `$autocapture` usage, but also other events), a breakdown of events by library used to send it, and a list of users who logged in during the time period in consideration. Open to more enhancements.